Mathlib/CategoryTheory/Limits/Preserves/Shapes/Biproducts.lean

```diff
@@ -84,6 +84,44 @@ def mapBinaryBicone {X Y : C} (b : BinaryBicone X Y) : BinaryBicone (F.obj X) (F
   inr_snd := by rw [← F.map_comp, b.inr_snd, F.map_id]
 #align category_theory.functor.map_binary_bicone CategoryTheory.Functor.mapBinaryBicone
 
+/-- An inverse to `CategoryTheory.Functor.mapBinaryBicone` -/
+@[simps]
+def mapBinaryBiconeInv [IsEquivalence F] {X Y : C} (b : BinaryBicone (F.obj X) (F.obj Y)) :
```

Comment:
I guess the "correct" way to do this would be to make `BinaryBicone` into a category and recreate the whole `Cones.functoriality` construction for `BinaryFan`, `BinaryCofan` and `BinaryBicone` and then relate these notions, but that would be quite a bit of work, so I wouldn't be opposed to adding this concrete definition for now if you have need for it. It is a bit sad though that right now we're not even able to state the relationship between `mapBinaryBicone` and `mapBinaryBiconeInv`.

Reply:
Ah, I just noticed that you're working towards this in #11102, nice.

Reply:
Indeed, but thanks for confirming that it is a sensible route to take!

Reply:
(I'm happy for this PR to wait on that one)